ci-operator/config/open-cluster-management/search-e2e-test/open-cluster-management-search-e2e-test-main.yaml

```diff
@@ -23,36 +23,11 @@ tests:
       IMAGE_REPO: search-e2e-test
       REGISTRY_ORG: open-cluster-management
     workflow: ocm-ci-image-mirror
-- as: lint
+- as: lint-unittest-sonarcloud
   commands: |
     export SELF="make -f Makefile.prow"
     make -f Makefile.prow install
     make -f Makefile.prow lint
-    make -f Makefile.prow check
-  container:
-    from: src
-- as: backplane-fast-forward
```

Comment:
Why did you get rid of the backplane fast-forward? I believe this means e2e test will no longer be fast-forwarded.